scene/animation/scene_tree_tween.cpp

```diff
@@ -0,0 +1,931 @@
+					f.basis.elements[2][1] - i.basis.elements[2][1],
+					f.basis.elements[2][2] - i.basis.elements[2][2],
+					f.origin.x - i.origin.x,
+					f.origin.y - i.origin.y,
+					f.origin.z - i.origin.z);
+		}
+
+		default: {
+			return Variant::evaluate(Variant::OP_SUBTRACT, p_final_val, p_intial_val);
+		}
+	};
+}
+
+void SceneTreeTween::_bind_methods() {
+	ClassDB::bind_method(D_METHOD("tween_property", "object", "property", "final_val", "duration"), &SceneTreeTween::tween_property);
+	ClassDB::bind_method(D_METHOD("tween_interval", "time"), &SceneTreeTween::tween_interval);
+	ClassDB::bind_method(D_METHOD("tween_callback", "object", "method", "binds"), &SceneTreeTween::tween_callback, DEFVAL(Array()));
+	ClassDB::bind_method(D_METHOD("tween_method", "object", "method", "from", "to", "duration", "binds"), &SceneTreeTween::tween_method, DEFVAL(Array()));
+
+	ClassDB::bind_method(D_METHOD("custom_step", "delta"), &SceneTreeTween::custom_step);
+	ClassDB::bind_method(D_METHOD("stop"), &SceneTreeTween::stop);
+	ClassDB::bind_method(D_METHOD("pause"), &SceneTreeTween::pause);
+	ClassDB::bind_method(D_METHOD("play"), &SceneTreeTween::play);
+	ClassDB::bind_method(D_METHOD("kill"), &SceneTreeTween::kill);
+	ClassDB::bind_method(D_METHOD("get_total_elapsed_time"), &SceneTreeTween::get_total_time);
+
+	ClassDB::bind_method(D_METHOD("is_running"), &SceneTreeTween::is_running);
+	ClassDB::bind_method(D_METHOD("is_valid"), &SceneTreeTween::is_valid);
+	ClassDB::bind_method(D_METHOD("bind_node", "node"), &SceneTreeTween::bind_node);
+	ClassDB::bind_method(D_METHOD("set_process_mode", "mode"), &SceneTreeTween::set_process_mode);
+	ClassDB::bind_method(D_METHOD("set_pause_mode", "mode"), &SceneTreeTween::set_pause_mode);
+
+	ClassDB::bind_method(D_METHOD("set_parallel", "parallel"), &SceneTreeTween::set_parallel, DEFVAL(true));
+	ClassDB::bind_method(D_METHOD("set_loops", "loops"), &SceneTreeTween::set_loops, DEFVAL(0));
+	ClassDB::bind_method(D_METHOD("set_speed_scale", "speed"), &SceneTreeTween::set_speed_scale);
+	ClassDB::bind_method(D_METHOD("set_trans", "trans"), &SceneTreeTween::set_trans);
+	ClassDB::bind_method(D_METHOD("set_ease", "ease"), &SceneTreeTween::set_ease);
+
+	ClassDB::bind_method(D_METHOD("parallel"), &SceneTreeTween::parallel);
+	ClassDB::bind_method(D_METHOD("chain"), &SceneTreeTween::chain);
+
+	ClassDB::bind_method(D_METHOD("interpolate_value", "initial_value", "delta_value", "elapsed_time", "duration", "trans_type", "ease_type"), &SceneTreeTween::interpolate_variant);
+
+	ADD_SIGNAL(MethodInfo("step_finished", PropertyInfo(Variant::INT, "idx")));
+	ADD_SIGNAL(MethodInfo("loop_finished", PropertyInfo(Variant::INT, "loop_count")));
+	ADD_SIGNAL(MethodInfo("finished"));
+
+	BIND_ENUM_CONSTANT(TWEEN_PAUSE_BOUND);
+	BIND_ENUM_CONSTANT(TWEEN_PAUSE_STOP);
+	BIND_ENUM_CONSTANT(TWEEN_PAUSE_PROCESS);
+}
+
+SceneTreeTween::SceneTreeTween() {
+	ERR_FAIL_MSG("Tween can't be created directly. Use create_tween() method.");
```

Comment:
If `interpolate_value` is non-static, you need to revert #60331 fully (including documentation changes for Tween etc)